docs/guide-using-an-ide-for-backend.md

```diff
@@ -1,21 +1,16 @@
-# Using an IDE (Intellij)
+# Using an IDE (Intellij) for development of MOLGENIS backend
```

Comment:
Actually, you can right-click the docker-compose.yml and press play to run the app services in IntelliJ for easier access to the logs and easier cleanup.